appveyor.yml

```diff
@@ -1,65 +1,86 @@
 version: '{branch}-{build}'
+build: off
 
 platform:
   - x86
   - x64
 
 environment:
   global:
-    PYTHONPATH: "C:\\testdir"
-    PYTHONWARNINGS: "ignore:::wheel.pep425tags:"
-    CONDA_BLD_VERSION: "3.5"
-    CONDA_BLD: "C:\\conda"
-    NUNIT: "nunit-console"
+    PYTHONUNBUFFERED: True
+    PYTHONWARNINGS: 'ignore:::wheel.pep425tags:'
+    PYTHONPATH: C:\testdir
+    NUNIT: nunit-console
+    CONDA_BLD: C:\conda
+    CONDA_BLD_VERSION: 3.5
 
     # SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the
     # /E:ON and /V:ON options are not enabled in the batch script interpreter
     # See: http://stackoverflow.com/a/13751649/163740
-    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci\\run_with_env.cmd"
+    CMD_IN_ENV: 'cmd /E:ON /V:ON /C .\ci\run_with_env.cmd'
 
   matrix:
-    - PYTHON_VERSION: "2.7"
-    - PYTHON_VERSION: "3.3"
-    - PYTHON_VERSION: "3.4"
-    - PYTHON_VERSION: "3.5"
-    - PYTHON_VERSION: "3.6"
+    - PYTHON_VERSION: 2.7
+    - PYTHON_VERSION: 3.3
+    - PYTHON_VERSION: 3.4
+    - PYTHON_VERSION: 3.5
+    - PYTHON_VERSION: 3.6
 
 init:
   # Prepare environment
   - mkdir C:\testdir
 
   # Set environment variables depending based on build cfg
-  - SET CONDA_PY=%PYTHON_VERSION:.=%
-  - SET CONDA_BLD_ARCH=%PLATFORM:x=%
-  - SET PYTHON=C:\PYTHON%CONDA_PY%
-  - IF %PLATFORM%==x86 (SET CONDA_BLD_ARCH=32)
-  - IF %PLATFORM%==x86 (SET NUNIT=%NUNIT%-x86)
-  - IF %PLATFORM%==x64 (SET PYTHON=%PYTHON%-x64)
+  - set CONDA_PY=%PYTHON_VERSION:.=%
+  - set CONDA_BLD_ARCH=%PLATFORM:x=%
+  - set PYTHON=C:\PYTHON%PYTHON_VERSION:.=%
+  - if %PLATFORM%==x86 (set CONDA_BLD_ARCH=32)
+  - if %PLATFORM%==x86 (set NUNIT=%NUNIT%-x86)
+  - if %PLATFORM%==x64 (set PYTHON=%PYTHON%-x64)
```

Comment:
`%CONDA_PY%` was removed originally, I added it back later, but decided to keep it on `PYTHON_VERSION` in case it was removed later.
The casing was to keep all constants on upper case and the rest on lowercase.